### ISSUE TITLE
Number of objects are 0 in validation set on precompute 

### ISSUE THREAD

**Issue by aminullah6264** [KuanchihHuang/MonoDTR#16]
./launchers/det_precompute.sh config/config.py train
Precomputation for the training/validation split
start reading training data
training split finished precomputing06s, eta:0.70s, **total_objs:[11670, 2156, 706], usable_objs:[11670, 2105, 700]]**
start reading validation data
**validation split finished precomputing01s, eta:0.90s, total_objs:[0, 0, 0], usable_objs:[0, 0, 0]]**
Preprocessing finished

What could be possibly wrong here?
